ui/pages/settings/contact-list-tab/add-contact/add-contact.component.js

```diff
@@ -134,6 +127,43 @@ export default class AddContact extends PureComponent {
               {t('ethereumPublicAddress')}
             </div>
             {this.renderInput()}
+            <div
+              className={`address-book__view-contact__group__${
+                domainResolutions?.length === 1 ? 'single-' : ''
+              }resolution-list`}
+            >
+              {domainResolutions?.length > 0 &&
+                domainResolutions.map((resolution) => {
+                  const {
+                    resolvedAddress,
+                    resolvingSnap,
+                    addressBookEntryName,
+                    protocol,
+                  } = resolution;
+                  const domainName = addressBookEntryName || this.state.input;
+                  return (
+                    <DomainInputResolutionCell
+                      key={`${resolvedAddress}${resolvingSnap}${protocol}`}
+                      domainType={
+                        protocol === 'Ethereum Name Service' ? 'ENS' : 'Other'
```

Comment:
Should ENS be translated?  I'm not sure...

Reply:
Going by our designs here: https://www.figma.com/design/7JOVfBCxfHiz8xOwbuWF3j/Snaps-Platform-v2?node-id=69-16627&t=CxxNxGwGA8BVPq1s-0


cc: @eriknson -- could use your comment on this part as well, I know @georgewrmarshall already tagged you about the icon size.